affected_purls:
- pkg:deb/debian/zodb@1:3.6.0-2%2Blenny3
- pkg:deb/debian/zodb@3.6.0-6
aliases:
- CVE-2010-3495
fixed_purl:
- pkg:deb/debian/zodb@1:3.10.7-1?distro=buster
- pkg:deb/debian/zodb@1:3.9.4-1.1
- pkg:deb/debian/zodb@1:3.9.4-1.1?distro=buster
references:
- id: 187004
  reference_id: ''
  url: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2010-3495
- id: 124297
  reference_id: '599711'
  url: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=599711
severities: []
summary: Race condition in ZEO/StorageServer.py in Zope Object Database (ZODB) before
  3.10.0 allows remote attackers to cause a denial of service (daemon outage) by establishing
  and then immediately closing a TCP connection, leading to the accept function having
  an unexpected return value of None, an unexpected value of None for the address,
  or an ECONNABORTED, EAGAIN, or EWOULDBLOCK error, a related issue to CVE-2010-3492.
vulnerability_id: VCID-1rpa-hdn4-aaas
weaknesses: []
